src/client/localplayer.cpp

```diff
@@ -433,12 +433,14 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 	}
 
 	/*
-		Check properties of the node on which the player is standing
+		Check properties of the node on which the player is standing or in which the player's feet
```

Comment:
I know you don't write English fluently so i suggest:

Check properties of the node stood on and the node above that containing the player's feet

And check the line is no longer than 90 columns (with tab size 4).

Reply:
`Check properties of the node the player stands on and the node above containing the player's feet`

...would probably be better, IMHO

Reply:
Yes that's slightly better, either is good.